src/deluge/gui/context_menu/configure_song_macros.cpp

```diff
@@ -83,11 +83,11 @@ ActionResult ConfigureSongMacros::padAction(int32_t x, int32_t y, int32_t on) {
 	if (sdRoutineLock) {
 		return ActionResult::REMIND_ME_OUTSIDE_CARD_ROUTINE;
 	}
-	// don't allow user to switch modes
```

Comment:
this looks wrong? Why is the behaviour changing so we can switch modes but not sections now